Source/Mosa.Compiler.Framework/Stages/SparseConditionalConstantPropagationStage.cs

```diff
@@ -49,6 +51,9 @@ protected override void Run()
 		if (HasProtectedRegions)
 			return;
 
+		Steps = 0;
+		MethodCompiler.CreateTranformInstructionTrace(this, Steps++);
```

Comment:
Is this intended to be 0 or 1 for the input parameter? Currently it'll input 0 as the increment happens afterwards.

Reply:
Should start at 0